### ISSUE TITLE
[RPR] Gluttony overrides active Gibbet and Gallows

### ISSUE THREAD

**Issue by StriKejk** [Nik-Potokar/XIVSlothCombo#1001]
Hi, like the title said I came across a bug where Gluttony would override an active Gibbet and Gallows attack and thus losing it in the process. This wasn't the case in earlier versions but I'm afraid I can not pinpoint the time it changed although it has been a couple of weeks at least.

This happens if you don't use everything on one button button instead use an extra button only for Gluttony/Blood Stalk/Gibbet & Gallows (and the Enshroud combo but that's not relevant).

In short: Have a separate button for Blood Stalk/Grim Swathe, activate the option to have it be overriden by Gluttony, also activate the option to have Gibbet & Gallows on the same button -> Active Gibbet & Gallows attack gets overriden by Gluttony and will be lost in the process.

Picture of relevant config attached:
![bug](https://user-images.githubusercontent.com/14365171/191383814-c2286f8a-36f8-4620-9b34-4d8c7dc0c0d6.jpg)

**Comment by StriKejk** [comment]
Issue still exists

**Comment by k-kz** [comment]
> Issue still exists

Ty for letting us know, pretty sure it was working in testing but we'll look into it.

Got some hotfixes coming soon so will try to get this resolved for that update.

**Comment by k-kz** [comment]
@StriKejk Hey, I misremembered the merge order when commenting before - this fix was implemented immediately after the current release (`v3.0.17.3`) and will show up in `v3.0.17.4`.

If you're still having any issues with this after the next update releases, leave a comment and we'll re-open and investigate further, but so far all good in testing 👍🏼
